tsl/test/isolation/expected/continuous_aggs_multi.out

```diff
@@ -119,11 +120,12 @@ step Refresh2: REFRESH MATERIALIZED VIEW continuous_view_2; <waiting ...>
 step UnlockCompleted: ROLLBACK;
 step Refresh2: <... completed>
 step UnlockMat1: ROLLBACK;
-R1: LOG:  materializing continuous aggregate public.continuous_view_1: nothing to invalidate, new range up to 30
+R1: LOG:  materializing continuous aggregate public.continuous_view_1: nothing to invalidate, new range up to 45
```

Comment:
Is this diff also related to the alter view fialing?

Reply:
Yup. Fixed.